gtda/homology/simplicial.py

```diff
@@ -71,9 +71,10 @@ class VietorisRipsPersistence(BaseEstimator, TransformerMixin, PlotterMixin):
         equals `coeff`.
 
     collapse_edges : bool, optional, default: ``False``
-        Whether to run the edge collapse algorithm in [2]_ prior to the
-        persistent homology computation (see the Notes). Can reduce the runtime
-        dramatically when the data or the maximum homology dimension are large.
+        Whether to run the edge collapse algorithm in Boissonnat & Pitram [2]_
```

Comment:
I'd prefer if we avoided author names. It would have to be done everywhere else for consistency, sometimes the list of authors is long, and I'd prefer if docstrings were not any longer than absolutely necessary. What do you think?

Reply:
Ok. Then we need to find another representation for a bibliographic entry: maybe `[2]`? That way, `[2] [2]_` should generate `[2]^2`, where the `^2` has a hyperlink.
What do you think?

Reply:
> Then we need to find another representation for a bibliographic entry

Sorry, I don't get this. Isn't this style exactly the same as elsewhere?

Reply:
If you write "(...) algorithm in [2]_ (...)", this will render as something like
![image](https://user-images.githubusercontent.com/32167173/92880574-6fc32180-f40e-11ea-899e-7c883e6529b7.png)
What i am suggesting consists of finding a representation "X" of the entry [2] (for example, X = Boissonnat & Pitram ), so that we say 
![image](https://user-images.githubusercontent.com/32167173/92880685-91240d80-f40e-11ea-8a03-d0d4810ac218.png)

Or, maybe, in this case, remove the "in" and put the `^2` after algorithm? Am I overthinking this?

Reply:
I'm confused as hell.